include/z64eff_blure.h

```diff
@@ -0,0 +1,95 @@
+#ifndef Z64EFF_BLURE_H
+#define Z64EFF_BLURE_H
+
+#include "ultra64.h"
+
+#include "color.h"
+#include "z64math.h"
+
+struct GraphicsContext;
+
+#define EFFECT_BLURE_COLOR_COUNT 4
+
+//#define EFFECT_BLURE_FLAG_1         (1 << 0)
+#define EFFECT_BLURE_FLAG_2         (1 << 1)
+#define EFFECT_BLURE_FLAG_4         (1 << 2)
+//#define EFFECT_BLURE_FLAG_8         (1 << 3)
+#define EFFECT_BLURE_FLAG_10        (1 << 4)
```

Comment:
Is there a reason flags 1 and 8 are commented out?

Reply:
Because they are not used and I forgot to remove them, whoops :sweat_smile: